espnet2/bin/s2st_inference.py

```diff
@@ -214,7 +215,7 @@ def __init__(
                 if st_subtask_token_type is None:
                     st_subtask_token_type = train_args.tgt_token_type
                 elif st_subtask_token_type == "bpe":
-                    if st_subtask_tokenizer is not None:
+                    if st_subtask_bpemodel is not None:
```

Comment:
Is it OK?
This changes some behavior.
If you confirmed it, it would be fine.

@ftshijt, please check it.

Reply:
Please let me know if this modification was correct.
However, the `st_subtask_tokenizer` is not declared at any line before L217, so `flake8` was raising errors of declaration.

Reply:
Thanks!
@ftshijt, please confirm it.